README.md

```diff
@@ -1,5 +1,5 @@
 Provides a service to interact with [Akismet](http://akismet.com)
-Can use [Buzz](https://github.com/kriswallsmith/Buzz) or [Guzzle](https://github.com/guzzle/guzzle) for the backend.
+Can use [Guzzle](https://github.com/guzzle/guzzle) for the backend.
```

Comment:
Maybe change the wording here to be "Uses Guzzle" instead of "Can use Guzzle". It seems like Guzzle is the only option now, right?